lib/destination/ddl/expiry_test.go

```diff
@@ -2,29 +2,43 @@ package ddl
 
 import (
 	"fmt"
+	"strings"
 	"testing"
 	"time"
 
+	"github.com/artie-labs/transfer/lib/config/constants"
+
 	"github.com/stretchr/testify/assert"
 )
 
 func TestShouldDeleteFromName(t *testing.T) {
-	tblsToNotDelete := []string{
-		"table", "table_", "table_abcdef9",
-		fmt.Sprintf("future_table_%d", time.Now().Add(1*time.Hour).Unix()),
-	}
-
-	for _, tblToNotDelete := range tblsToNotDelete {
-		assert.False(t, ShouldDeleteFromName(tblToNotDelete), tblToNotDelete)
-	}
+	{
+		// Tables to not drop
+		tablesToNotDrop := []string{
+			"foo",
+			"transactions",
+			fmt.Sprintf("expired_tbl__artie_suffix_%d", time.Now().Add(constants.TemporaryTableTTL).Unix()),
```

Comment:
Call this `future_tbl__...` for clarity?

Reply:
Yup!